fsd_utils/config/commonconfig.py

```diff
@@ -0,0 +1,130 @@
+import logging
+import os
+from pathlib import Path
+
+class CommonConfig:
+
+    # ---------------
+    #  General App Config
+    # ---------------
+    SECRET_KEY = os.getenv("SECRET_KEY", "secret_key")
+    SESSION_COOKIE_NAME = os.getenv("SESSION_COOKIE_NAME", "session_cookie")
+    FLASK_ENV = os.getenv("FLASK_ENV", "development")
```

Comment:
Just for discussion maybe, should we have a default for FLASK_ENV or should this be something we require to be explicitly set?

Reply:
Agree - will change to throw a key error when flask_env not set as per chat